raster/r.horizon/testsuite/test_r_horizon.py

```diff
@@ -215,6 +215,7 @@ def test_point_mode_multiple_direction_json(self):
         reference["y"] = 216180.0
         reference["azimuth"] = azimuths
         reference["horizon_height"] = horizons
+        stdout[0].pop("horizon_distance", None)
```

Comment:
If I understand correctly, reverting the code change would still keep the test as passing. In other words, this does not test the expectations. We should test that distance is emitted by the tool.